x/pylons/keeper/pylons_account.go

```diff
@@ -15,9 +15,8 @@ func (k Keeper) SetPylonsAccount(ctx sdk.Context, accountAddr types.AccountAddr,
 	binaryUsername := k.cdc.MustMarshal(&username)
 	usernamePrefixStore := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.UsernameKey))
 	accountPrefixStore := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.AccountKey))
-
-	usernamePrefixStore.Set(types.KeyPrefix(username.Value), binaryUsername)
-	accountPrefixStore.Set(types.KeyPrefix(accountAddr.Value), binaryAddr)
+	usernamePrefixStore.Set(types.KeyPrefix(username.Value), binaryAddr)
```

Comment:
what is the purpose of this reverting

Reply:
Because the change of code before doesn't match with the logic of code so I have to revert it 
https://github.com/Pylons-tech/pylons/blob/7850eb359074d074a90df5fc87ace9580d5e1a83/x/pylons/keeper/pylons_account.go#L10-L12

Reply:
It's the reason why we can pass the test in github's workflow